src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -32,28 +32,22 @@
 
 DATA_SAVE_ID  = wx.NewId()
 
-
 class ExtendedToolbar(NavToolbar):
     """An extended plotting toolbar with a save and close button."""
 
     def __init__(self, canvas):
+        # override class var to exclude subplots
+        self.toolitems = tuple(el for el in NavToolbar.toolitems\
+                if el[0] != 'Subplots')
         NavToolbar.__init__(self, canvas)
-        # Get rid of the configure subplots button
-        if hasattr(self, 'wx_ids'):
-            self.DeleteTool(self.wx_ids['Subplots'])
-        else:
-            self.DeleteToolByPos(6)
         # Add new buttons
         self.AddSimpleTool(wx.ID_PRINT,
                wx.ArtProvider.GetBitmap(wx.ART_PRINT, wx.ART_TOOLBAR),
                'Print', 'print graph')
         self.AddSimpleTool(DATA_SAVE_ID,
-               _load_bitmap('stock_save_as.xpm'),
+               _load_bitmap('filesave.png'),
```

Comment:
This produces two same-image icons with different effects and still uses non-API MPL function `_load_bitmap`.

Please grab the "stock_save_as.xpm" from matplotlib repo prior to matplotlib/matplotlib#5626, convert it to PNG and add it to the /icons directory in PDFgui repo.  Then use the `diffpy.pdfgui.gui.pdfguiglobals.iconpath` function instead of `_load_bitmap`; you might also need `wx.Bitmap` call to load the PNG image.

Please test the toolbar look with both matplotlib 1 and matplotlib 2.  If either looks weird, we may need 2 icon files - a colorful version for MPL-1 and a black-and-white for MPL-2.